Code/Editor/CryEdit.cpp

```diff
@@ -1352,8 +1352,27 @@ void CCryEditApp::CompileCriticalAssets() const
         }
     }
     assetsInQueueNotifcation.BusDisconnect();
-    CCryEditApp::OutputStartupMessage(QString("Asset Processor is now ready."));
 
+    // Signal the "CriticalAssetsCompiled" lifecycle event
+    // Also reload the "assetcatalog.xml" if it exists
+    if (auto settingsRegistry = AZ::SettingsRegistry::Get(); settingsRegistry != nullptr)
+    {
+        AZ::ComponentApplicationLifecycle::SignalEvent(*settingsRegistry, "CriticalAssetsCompiled", R"({})");
```

Comment:
[nit] the language used around AssetProcessor is "to process assets". To keep that in line with the event, maybe the event could be called "CriticalAssetsProcessed"?

Reply:
Well the existing functions are already called `CriticalAssetsCompiled` so their is existing precedence for that as well

Reply:
That's ok